### PR TITLE
[Merged by Bors] - chore: add json-sql to repo

### PR DESCRIPTION
**Pull request by ozgrakkurt**
Copy-pasted from the archived repo: https://github.com/infinyon/fluvio-connectors.

I made a small modification to `crates/json-sql/src/transform.rs:29` to make it compile withthe current version of `fluvio-model-sql`

### CODE REVIEW

File: crates/json-sql/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# JSON to SQL transformation SmartModule
+This is a `map` type SmartModule that converts records in arbitrary JSON into
+records in [SQL model](../../rust-connectors/models/fluvio-model-sql). The transformation specification is defined in
```
Comment (galibey) [inline crates/json-sql/README.md:3]:
```suggestion
records in [SQL model](../fluvio-model-sql). The transformation specification is defined in 
```

File: crates/json-sql/README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+| device.number    | 1                                                            |
+| $.device.number  | 1                                                            |
+| .device.number   | 1                                                            |
+| device.metrics   | ["one", "two", "three"]                                      |
+| device.metrics.0 | "one"                                                        |
+| device.metrics.1 | "two"                                                        |
+| $                | {"device":{"number": 1}, "metrics": ["one", "two", "three"]} |
+
+### Data types
+The list of supported types and corresponding types from [SQL model](../../rust-connectors/models/fluvio-model-sql):
```
Comment (galibey) [inline crates/json-sql/README.md:95]:
```suggestion
The list of supported types and corresponding types from [SQL model](../fluvio-model-sql):
```

**Review by galibey — APPROVED** [review]
LGTM. Only need to update links in README file

**Comment by ozgrakkurt** [comment]
> LGTM. Only need to update links in README file

thanks!

**Pull request closed without merging**